packages/spacex_api/lib/src/models/launch.dart

```diff
@@ -0,0 +1,150 @@
+import 'package:equatable/equatable.dart';
+import 'package:json_annotation/json_annotation.dart';
+import 'package:spacex_api/spacex_api.dart';
+
+part 'launch.g.dart';
+
+/// {@template launch}
+/// A model containing data about a launch
+/// {@endtemplate}
+@JsonSerializable()
+class Launch extends Equatable {
+  /// {@macro launch}
+  const Launch({
+    required this.id,
+    required this.name,
+    required this.links,
+    this.details,
+    this.crew,
+    this.flightNumber,
+    this.rocket,
+    this.success,
+    this.dateUtc,
+    this.dateLocal,
+  });
+
+  /// The ID of the launch.
+  final String id;
+
+  /// The name of the launch.
+  final String name;
+
+  /// The details of the launch.
+  ///
+  /// May be null
+  final String? details;
+
+  /// A List of crew members
+  ///
+  /// May be empty.
+  final List<CrewMember>? crew;
+
+  /// The flightNumber of the launch
+  final int? flightNumber;
+
+  /// The ID of the rocket
+  final String? rocket;
+
+  /// If launch succeeded
+  final bool? success;
+
+  /// The launch date in UTC
+  final DateTime? dateUtc;
+
+  /// The launch date
+  final DateTime? dateLocal;
+
+  /// Available source links
+  final Links links;
+
+  @override
+  List<Object?> get props => [
+        id,
+        name,
+        details,
+        crew,
+        flightNumber,
+        rocket,
+        success,
+        dateUtc,
+        dateLocal
+      ];
+
+  /// Converts a JSON [Map] into a [Launch] instance
+  static Launch fromJson(Map<String, dynamic> json) => _$LaunchFromJson(json);
+
+  /// Converts this [Launch] instance into a JSON [Map]
+  Map<String, dynamic> toJson() => _$LaunchToJson(this);
+
+  @override
+  String toString() => 'Latest Launch($id, $name)';
+}
+
+/// {@template links}
+/// A model that represents available links to images, videos and articles.
+/// {@endtemplate}
+@JsonSerializable(
+  fieldRename: FieldRename.snake,
+)
+class Links extends Equatable {
+  /// {@macro length}
```

Comment:
```suggestion
  /// {@macro links}
```

For convenience, I'd recommend doing a search for all `@template` and `@macro` calls and see if they match 👍🏻 
I mess these up all the time, too 😋